FAQ 2.10.1 (#233)
firebirdberlin/nightdream: docs/faq/faq_en.md

Using a two-finger pinch gesture you can freely resize the clock layout.

### How can I change the position of the clock ?

With a long press on the clock a frame is displayed and the clock can be freely positioned on the screen.
The position is automatically saved as soon as the clock is released.

### The clock automatically changes its position 

The app offers two different options for screen protection:
- move the clock around
- fade the clock

> If an automatic change of position of the clock is not desired, the screen protection can be changed 

Reviewer: ```suggestion
If an automatic change of position of the clock is not desired, the screen protection mode can be changed 
```

### Which pictures are shown in the slideshow

The app shows all images in JPG format.
Videos or images in any other format are not displayed.
It is currently not possible to select individual images for the slideshow.

> ### How can I select images

Reviewer: ```suggestion
### How can I select images?
```

> ### How can I choose a selection of images

Reviewer: ```suggestion
### How can I select images?
```

> For Android Lollipop (version 5.0) and above a folder can be selected for display.

Reviewer: ```suggestion
For Android Lollipop (version 5.0) and above a folder can be selected for displaying images.
```

It is currently not possible to select individual images for the slide show.
It is recommended to create your own folder and copy the images you like for the slide show into this folder.
For Android Versions below Lollipop the device's photo directory is selected automatically.

### The slide show appears with a delay at app startup

The loading of the individual images takes place in the background to ensure smooth operation of the app.
On older devices or large images it may take some time before the image is ready to be displayed.
In this case please try to reduce the size of the problematic image.

### How can cloud storage be accessed?

It is currently not possible access to any cloud storage.

### Only a black screen is displayed

Please check if there are pictures in JPG format on the device. 
The option * Hide the background image in night mode * may be activated. 
If so, a black background is shown instead of a slideshow during the night.

> ### EXIF informations are not displayed

Reviewer: ```suggestion
### EXIF information is not displayed
```

Please check whether EXIF information is included in the displayed JPG image.
To determine the location from the EXIF information the Google Play Services are required.
Some Android distributions may not have these preinstalled.

## Speak the current time

### The current time is not read out

The app requires an installed text-to-speech engine (TTS) to read the current time.
Please check in the device settings whether this is installed and activated on your device.

### How can the language be changed?

> The selected language depends on the language of your device and cannot be changed in the app.

Reviewer: ```suggestion
The selected language depends on the language of your device and cannot be changed within the app.
```

## Notifications

### What is the media control?

The media control makes it possible to take control of external music players which are controlled 
through the notification center.

### How do I activate the media control?

> The media control can be activated in the app under settings / notifications.

Reviewer: ```suggestion
The media control notifications can be activated in the app preferences in the  notifications section.
```
